cli/waiter/cli.py

```diff
@@ -44,6 +44,12 @@
     'ssh': {
         'run-function': ssh.register(subparsers.add_parser)
     },
+    'start': {
```

Comment:
If a user runs `waiter start` on a token that isn't in maintenance mode then the token won't get pinged (service won't be started), which can be confusing for the user. I think the behavior here should be user runs `waiter start`
1. if token is in maintenance mode then disable maintenance mode
2. always ping token

Reply:
we decided to make this a separate follow up PR